src/Symfony/Component/Cache/StatsInterface.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Cache;
+
+/**
+ * Subset of the Doctrine\Common\Cache\Cache interface.
+ */
+interface StatsInterface
+{
+    const STATS_HITS = 'hits';
```

Comment:
I know I'm a bit obsessed with constant names ... but these names seem a bit redundant to me:

```php
StatsInterface::STATS_HITS
StatsInterface::STATS_UPTIME
StatsInterface::STATS_MEMORY_AVAILABLE
// ...
```

What about this instead?

```php
StatsInterface::HITS
StatsInterface::UPTIME
StatsInterface::MEMORY_AVAILABLE
// ...
```